lib/src/progress-bar/ProgressBar.tsx

```diff
@@ -66,19 +88,9 @@ const BackgroundProgressBar = styled.div`
   width: 100%;
 `;
 
-const DXCProgressBar = styled.div`
+const DXCProgressBar = styled.div<ProgressBarPropsType>`
```

Comment:
The problem here is because of the complexity of the `margin` prop:

![image](https://user-images.githubusercontent.com/34685461/182350652-10d9667d-468f-4955-bfe2-38caa8350708.png)
![image](https://user-images.githubusercontent.com/34685461/182350705-c21668f8-bfc7-4528-a2ff-7a3a101175e2.png)

So I am not so sure if it will be good a solution redoing the types of this prop.

Reply:
You can export both types. You can see an example of this in Link types:
```
export type Space = "xxsmall" | "xsmall" | "small" | "medium" | "large" | "xlarge" | "xxlarge";
export type Margin = {
  top?: Space;
  ...
};
```

Then:
`import { Margin, LinkProps, Space } from "./types";`

Reply:
Done